src/content/docs/solvro/Statute.mdx

```diff
@@ -121,6 +121,50 @@ description: Regulamin KN Solvro
 
 ---
 
+## § X Sekcje
+
+1. Sekcja jest organem działającym bezpośrednio pod Zarządem w określonym obszarze merytorycznym, komplementarnym do głównych celów i zadań KN Solvro.
+2. Głównym celem sekcji jest podejmowanie działań wspierających realizację misji i wizji Koła, a także rozwój kompetencji i zaangażowania jej członków.
+3. Sekcja jest powoływana bądź rozwiązywana w drodze:  
+   3.1. Uchwały Zarządu.  
+   3.2. Uchwały Walnego Zgromadzenia.
+
+4. Na czele sekcji stoi Przewodniczący sekcji, który jest Członkiem Aktywnym KN Solvro.
+
+5. Przewodniczący sekcji jest wybierany:  
+   5.1 Uchwałą Zarządu Walnego Zgromadzenia.  
+   5.2 Uchwałą Walnego Zgromadzenia.
+
+6. Przewodniczący sekcji ponosi odpowiedzialność za realizację zadań sekcji oraz za zapewnienie zgodności jej działań z postanowieniami Regulaminu, uchwałami Walnego Zgromadzenia, planem Taktycznym i decyzjami Zarządu.
+
+7. Kompetencje sekcji  
```

Comment:
ogólnie polecam numerować w markdownie, wtedy nie musisz pilnować numerków

1.
1.
1.

Reply:
![image](https://github.com/user-attachments/assets/c4257a97-8b95-41ae-9299-b619feb788f6)

chodzi, że możesz pisać w ten sposób i potem markdown sam sobie ogarnie, niestety github tez to ogarnął

Reply:
żebyś nie musiał ręcznie pilnować numerków